Example/Controllers/Extensions/ViewController+Activity.swift

```diff
@@ -0,0 +1,6 @@
+import Foundation
+import UIKit
+
+public class MiniAppActivityController: UIActivityViewController {
+    public override func dismiss(animated flag: Bool, completion: (() -> Void)? = nil) {}
```

Comment:
So we don't need to call `super` here 🤔

Reply:
that's the fix it doesn't dimiss anything but the activity vc (works like a joy)